app/models/metric_rollup.rb

```diff
@@ -47,4 +47,12 @@ def extract_from_min_max(col)
       return val
     end
   end
+
+  def self.get_latest_hourly_rollups(resource_type, resource_ids)
+    metrics = where(:resource_type => resource_type, :capture_interval_name => "hourly")
+    metrics = metrics.where(:resource_id => resource_ids) if resource_ids
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.select('DISTINCT ON (metric_rollups.resource_id) metric_rollups.*')
+    metrics.includes(:resource)
+  end
```

Comment:
sorry for leaving this dangling. hard to articulate my thoughts here.

Are we able to make this less specific to this method?
Need to double check this query - don't think it will touch the indexes either